### PR TITLE
Bug 1315933 - Update npm dependencies.

### PR DESCRIPTION
**Pull request by jgraham**
This is required because the older version of grunt-angular-templates
was hanging with component templates. In order to upgrade everything it
was also necessary to switch from grunt-cache-busting, which is no longer
maintained, to grunt-cache-bust, which is.

---
This change is [Reviewable](https://reviewable.io/reviews/mozilla/treeherder/1968)

### CODE REVIEW

**Comment by autolander** [comment]
Autolander could not find a bug number in your pull request title. All pull requests should be in the format of: Bug [number] - [description].

**Comment by edmorley** [comment]
The shrinkwrap will also need updating (steps in RTD).
Looking forward to when we can switch to Yarn and reduce the pain of it further.

Thank you for updating these deps!

File: .eslintrc [hunk 1]
```diff
@@ -21,7 +21,7 @@
         "no-caller": 2,
         "no-div-regex": 2,
         "no-else-return": 2,
-        "no-empty-label": 2,
+        "no-labels": 2,
```
Comment (edmorley) [inline .eslintrc]: These are alphabetical, so maybe leave this entry lower down? :-)

File: .travis.yml [hunk 2]
```diff
@@ -11,25 +11,22 @@ matrix:
...
         # Create a clean virtualenv rather than using the one given to us,
         # to work around: https://github.com/travis-ci/travis-ci/issues/4873
         - if [[ ! -f ~/venv/bin/activate ]]; then virtualenv -p python ~/venv; fi
         - source ~/venv/bin/activate
         - pip install --disable-pip-version-check --upgrade pip==8.1.1
       install:
-        - npm install
         - pip install --disable-pip-version-check --require-hashes -r requirements/common.txt -r requirements/dev.txt
       script:
-        - grunt checkjs
```
Comment (edmorley) [inline .travis.yml]: Good call - I thought about doing this a couple of times :-)

**Pull request merged**